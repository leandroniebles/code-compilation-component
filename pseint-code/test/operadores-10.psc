Proceso sin_titulo
	Escribir Verdadero y 42;
	Escribir Verdadero o 42;
	Escribir 42 y Verdadero;
	Escribir 42 o Verdadero;
FinProceso
