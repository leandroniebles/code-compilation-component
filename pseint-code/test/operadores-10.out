=== Line 2: SynError 289
=== Line 3: SynError 289
=== Line 4: SynError 289
=== Line 5: SynError 289
